content/posts/expiring-auth-tokens.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Breaking Changes to Ecosounds & A2O Authentication"
+date: 2025-03-14T14:20:00+10:00
+image: "/images/ecosounds-banner-image.jpg"
+imageCredit: "QUT Ecoacoustics Staff"
+tags:
+    - News
+    - Technical
+---
+
+The Acoustic Workbench software that powers sites such as [Ecosounds](https://www.ecosounds.org/) and
+[A2O](https://data.acousticobservatory.org/) has been upgraded with enhanced security features.
+These changes will break existing workflows and automated scripts that depend on authentication token.
+
+## Effected Users
```

Comment:
Affected

Reply:
And following uses

Reply:
What was the purpose of the "following" word here?

It doesn't seem to flow in my mind

Wouldn't "Affected Users and Uses" flow a bit better than "Affected Users and Following Uses"